meta-ostro/recipes-image/images/ostro-image.bb

```diff
@@ -190,9 +190,16 @@ ostro_root_authorized_keys () {
 # Do not create ISO images by default, only HDDIMG will be created (if it gets created at all).
 NOISO = "1"
 
+# Add support for compressing images with zip. Works for arbitrary image
+# types. Example: OSTRO_VM_IMAGE_TYPES = "dsk.zip dsk.vdi.zip"
+COMPRESSIONTYPES_append = " zip"
+ZIP_COMPRESSION_LEVEL ?= "-9"
+COMPRESS_CMD_zip = "zip ${ZIP_COMPRESSION_LEVEL} ${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.${type}.zip ${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.${type}"
+COMPRESS_DEPENDS_zip = "zip-native"
+
 # Replace the default "live" (aka HDDIMG) images with whole-disk images
 # XXX Drop the VM hack after taking care also of the non UEFI devices (those using U-Boot: edison and beaglebone)
-OSTRO_VM_IMAGE_TYPES ?= "dsk dsk_xz dsk.vdi"
+OSTRO_VM_IMAGE_TYPES ?= "dsk dsk.xz dsk.vdi"
 IMAGE_FSTYPES_remove_intel-core2-32 = "live"
```

Comment:
What I hadn't considered is that disabling compression in this PR would also disable compression in the CI.

Let's deal with changing the default in a separate PR. We need to set up the CI first such that it includes a certain .inc file from meta-ostro, then in that other PR populate the CI settings while changing the non-CI defaults.

@kad: Okay to keep this PR as it is?